packages/teleport/src/lib/tdp/client.ts

```diff
@@ -149,6 +154,24 @@ export default class Client extends EventEmitter {
     );
   }
 
+  handleMfaChallenge(buffer: ArrayBuffer) {
+    const mfaJson = this.codec.decodeMfaJson(buffer);
```

Comment:
If this can throw an error as outlined in the comment on line 162 then shouldn't this be wrapped in a try/catch so that it can fail gracefully?